### PR TITLE
[ADD] fetch openupgrade 10.0 docs too

### PR DESCRIPTION
**Pull request by hbrunn**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by hbrunn** [comment]
the script pulls the default branch btw, should we change the default to 11 by now?

**Comment by pedrobaeza** [comment]
Yes, let's change the default branch at this stage.  Should we freeze the branch to be cloned on the scripts for avoiding errors on new releases until it's manually fixed?

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by hbrunn** [comment]
better not, then we don't notice the errors. Note that building the docs is part of the tests

**Comment by pedrobaeza** [comment]
Default branch changed.  Waiting for Travis result for merging.

**Comment by hbrunn** [comment]
the error here is what we want, this turns green when oca/openupgradelib#83 is merged

**Comment by hbrunn** [comment]
restarted build after @pedrobaeza merged the PR in openupgradelib

**Comment by pedrobaeza** [comment]
Restarting Travis now that the other PR has been merged.

**Comment by pedrobaeza** [comment]
hahaha, we are both the quickest gunman on this side of Odoo

**Comment by pedrobaeza** [comment]
Can you squash both commits?

File: scripts/build_openupgrade_docs [hunk 1]
```diff
@@ -6,7 +6,8 @@ DOC_PARTS="7.0/openerp/openupgrade/doc/source/modules50-60.rst
 7.0/openerp/openupgrade/doc/source/modules60-61.rst
 7.0/openerp/openupgrade/doc/source/modules61-70.rst
 8.0/openerp/openupgrade/doc/source/modules70-80.rst
-9.0/openerp/openupgrade/doc/source/modules80-90.rst"
+9.0/openerp/openupgrade/doc/source/modules80-90.rst
+10.0/odoo/openupgrade/doc/source/modules90-100.rst"
```
Comment (pedrobaeza) [inline scripts/build_openupgrade_docs]: Can we put the quote in other line for avoiding 2 lines diff on each version?

**Review by StefanRijnhart — APPROVED** [review]

File: scripts/build_openupgrade_docs (hunk 1 above)
Comment (StefanRijnhart) [reply scripts/build_openupgrade_docs]: Think optimistic ;-) The optimistic way is to merge and then create a new PR yourself

Comment (pedrobaeza) [reply scripts/build_openupgrade_docs]: Well, I don't know very well this part. That's why I'm asking if it can be done.

Comment (hbrunn) [reply scripts/build_openupgrade_docs]: added

**Comment by hbrunn** [comment]
I think it would be conceptually wrong to squash those two commits as they do very different things

**Comment by pedrobaeza** [comment]
Thanks, merging then with 2 commits.

**Pull request merged**